test/quantization/test_quantize_fx.py

```diff
@@ -0,0 +1,116 @@
+import torch
+import torch.nn.functional as F
+
+# symbolic trace
+from torch.fx import symbolic_trace
+
+# graph mode quantization based on fx
+from torch.quantization._quantize_fx import (
+    Quantizer,
+    fuse,
+)
+
+# eager mode quantization
+from torch.quantization import default_qconfig
+
+# test utils
+from torch.testing._internal.common_quantization import (
+    QuantizationTestCase,
+)
+
+class TestQuantizeFx(QuantizationTestCase):
+    def test_functional(self):
```

Comment:
would be awesome to also have a basic QAT test case

Reply:
currently I only have integration tests with torchvision, plan to add it after ddp test works. 
also it will be added in the future when we add individual op tests as well.